optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc

```diff
@@ -476,6 +476,46 @@ Use a lower tabu size than in a pure Tabu Search configuration.
   </localSearch>
 ----
 
+[[greatDeluge]]
+== Great Deluge
+
+
+[[greatDelugeAlgorithm]]
+=== Algorithm Description
+
+Great Deluge algorithm is very similar to the Simulated Annealing algorithm, it evaluates only a few moves per steps,
+so it steps quickly. The first accepted move is the winning step. A move is accepted only if it does not decrease
```

Comment:
"decrease" -> "is lower than"

Those that aren't lower than the water level, don't decrease it either.